airbyte-commons/src/main/java/io/airbyte/commons/features/EnvVariableFeatureFlags.java

```diff
@@ -23,4 +23,9 @@ public boolean autoDisablesFailingConnections() {
     return Boolean.parseBoolean(System.getenv("AUTO_DISABLE_FAILING_CONNECTIONS"));
   }
 
+  @Override
```

Comment:
why do we have a separate class for `EnvVariableFeatureFlags` from `EnvConfigs`? Doesn't have to change now, but not clear to me why they are separate.

Reply:
It was to potentially support a change to a DB.